app/helpers/categories_helper.rb

```diff
@@ -0,0 +1,2 @@
+module CategoriesHelper
+end
```

Comment:
Helper files that don't have anything in them can also be left out of git like with empty javascript and css files.